src/components/CommandBar/CommandBarReview.tsx

```diff
@@ -57,19 +60,75 @@ function CommandBarReview({ stepBack }: { stepBack: () => void }) {
     })
   }
 
+  const availableOptionalArgs = useMemo(() => {
+    if (!selectedCommand?.args) return undefined
+    const s = { ...selectedCommand.args }
+    for (const [name, arg] of Object.entries(s)) {
+      const value =
+        (typeof argumentsToSubmit[name] === 'function'
+          ? argumentsToSubmit[name](commandBarState.context)
+          : argumentsToSubmit[name]) || ''
+      const isHidden =
+        typeof arg.hidden === 'function'
+          ? arg.hidden(commandBarState.context)
+          : arg.hidden
+      const isRequired =
+        typeof arg.required === 'function'
+          ? arg.required(commandBarState.context)
+          : arg.required
+      if (isHidden || isRequired || value) {
+        delete s[name]
+      }
+    }
+    return s
+  }, [selectedCommand])
   return (
-    <CommandBarHeader>
-      <p className="px-4 pb-2">
-        {selectedCommand?.reviewMessage ? (
-          selectedCommand.reviewMessage instanceof Function ? (
-            selectedCommand.reviewMessage(commandBarState.context)
-          ) : (
-            selectedCommand.reviewMessage
-          )
-        ) : (
-          <>Confirm {selectedCommand?.displayName || selectedCommand?.name}</>
-        )}
-      </p>
+    <CommandBarHeader stepBack={stepBack}>
+      {selectedCommand?.reviewMessage && (
+        <>
+          <p className="px-4 py-2">
+            {selectedCommand.reviewMessage instanceof Function
+              ? selectedCommand.reviewMessage(commandBarState.context)
+              : selectedCommand.reviewMessage}
+          </p>
+          <CommandBarDivider />
+        </>
+      )}
+      {Object.entries(availableOptionalArgs || {}).length > 0 && (
+        <>
+          <div className="px-4">
+            <p className="mb-2">Optional arguments</p>
+            <div className="text-sm flex gap-4 items-start">
+              <div className="flex flex-1 flex-wrap gap-2">
```

Comment:
Typically you should be able to collapse these into one div if the outer one is just wrapping one child, but I'll have to double-check I might be missing something on my phone

Reply:
Oh yes thanks, made the change with the other one about inlining!